Lection 3/task 3.1/ReportApp/Models/Entity/Person.cs

```diff
@@ -1,11 +1,18 @@
-﻿namespace ReportApp.Models;
+﻿using ReportApp.Interfaces;
 
-public class Client : Person
+namespace ReportApp.Models.Entity;
+
+public class Person : IPerson
 {
```

Comment:
Please, choose one of the variants. Either you wanna keep it like an interface - then DRY, you wanna keep it as single model - remove interface, overcomplicated for nothing